docs/development.md

```diff
@@ -84,7 +94,11 @@ guide you through the process.
 
 ## Start the App
 
-Make sure that PostgreSQL is running.
+Make sure that PostgreSQL is running. If you have installed via Homebrew you can do this with the following command:
+
+```bash
+postgres -D /usr/local/var/postgres
```

Comment:
this was the thing I don't love... can you use brew services? i.e. `brew services start postgres`?